jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -133,6 +133,24 @@ def test_sort(self, harness: primitive_harness.Harness):
       raise unittest.SkipTest("GPU tests are running TF on CPU")
     self.ConvertAndCompare(harness.dyn_fun, *harness.dyn_args_maker(self.rng()))
 
+  @primitive_harness.parameterized(primitive_harness.lax_linalg_qr)
+  def test_qr(self, harness: primitive_harness.Harness):
+    # See jax.lib.lapack.geqrf for the list of compatible types
+    if not harness.params["dtype"] in [jnp.float32, jnp.float64, jnp.complex64, jnp.complex128]:
+      with self.assertRaisesRegex(NotImplementedError, "Unsupported dtype"):
```

Comment:
A small detail: I think this code would be a tiny bit more readable if you reorder the conditionals:
```
if dtype in [float32, float64]:
    ...
elif dtype in [complex64, complex128]:
   ...
else:
   assertraises unsupported dtype
```

Reply:
Addressed in latest commits